docs/feature_flags.md

```diff
@@ -183,4 +183,26 @@ This state is periodically ([`max_stale`][d2c]) cleared of inactive series.
 Enabling this _can_ have negative impact on performance, because the in-memory
 state is mutex guarded. Cumulative-only OTLP requests are not affected.
 
+### PromQL arithmetic expressions in time durations
+
+`--enable-feature=promql-duration-expr`
+
+With this flag, arithmetic expressions can also be used in time durations. The following operators are supported:
```

Comment:
I think we should list where this happens, or describe it somehow to not burden the reader. Maybe give some examples for range query and offset and mention that it's not supported for "@" operator. Interestingly arithmetic expressions on number values already worked, like "2m + 2m" is valid PromQL and gives you 240.

Reply:
what's the resolution?